File: 백준/Bronze/1978. 소수 찾기/소수 찾기.js
const fs = require('fs');
const inputs = fs.readFileSync('/dev/stdin').toString().trim().split('\n');

const nums = inputs[1].split(' ').map(Number);

let cnt = 0;
for(let n of nums) {
    let isPrime = true;

    if (n === 1) continue;
    for (let i = 2; i < n; i++) {
        if (n <= 1 || n % i === 0) {
            isPrime = false;
            break;
        }
    }

    if (isPrime) cnt++;
}

console.log(cnt);

File: 백준/Bronze/1978. 소수 찾기/소수 찾기.py
n = int(input())
arr = list(map(int, input().split()))

def isPrime(num):
    if num == 1: return 0

    for i in range(2, num + 1):
        if i * i > num: continue
        if num % i == 0: return 0

    return 1

cnt = 0
for a in arr:
    if isPrime(a): cnt += 1

print(cnt)
